InfluxClient.Tests/LineProtocolTests.cs

```diff
@@ -73,5 +73,30 @@ public void Format_WithTagContainingEqualSign_IsSuccessful()
             Assert.AreEqual<string>(expectedFormat, retval);
         }
 
+        [TestMethod]
+        [TestCategory("Format")]
+        public void Float64bit()
+        {
+            // Make sure correct decimal symbol is used in test case
+            System.Globalization.CultureInfo.DefaultThreadCurrentCulture = System.Globalization.CultureInfo.InvariantCulture;
```

Comment:
You may want to update [LineProtocol] to not use the system default, but that would be a separate issue.